rsconnect_jupyter/static/rsconnect.js

```diff
@@ -283,7 +283,8 @@ define([
                             if (result['finished']) {
                                 if (result['code'] != 0) {
                                     var msg = 'Failed to deploy successfully: ' + result['error'];
-                                    addValidationMarkup(false, $deploy_err, msg);
+                                    $deploy_err.closest(".form-group").addClass("has-error");
```

Comment:
I'd add a `TODO` here- this really belongs in the rejection handler when it's called in `connect.js`, but this fixes the issue.